src/textual/keys.py

```diff
@@ -256,3 +256,14 @@ def _get_key_display(key: str) -> str:
     if unicode_character.isprintable():
         return unicode_character
     return upper_original
+
+
+def _character_to_key(character: str) -> str:
+    """Convert a single character to a key value."""
+    assert len(character) == 1
+    if not character.isalnum():
+        key = unicodedata.name(character).lower().replace("-", "_").replace(" ", "_")
```

Comment:
Using `str.translate` with a pre-built table might be faster 🤞

Reply:
True! But doubt it would make a material difference in this case, with such small strings.